.github/workflows/tests.yml

```diff
@@ -102,255 +107,49 @@ jobs:
-          for /d %%F in (%UserProfile%\cbtmp_serial\*) do rd /s /q "%%F"
-          for /d %%F in (%UserProfile%\cbtmp\*) do rd /s /q "%%F"
-          call conda create -n blarg -yq --download-only python=2.7||exit 1
-          call conda create -n blarg -yq --download-only python=3.8||exit 1
-          call conda create -n blarg -yq --download-only python cmake||exit 1
-          mkdir ${{ github.workspace }}\\pytest-replay
-          set "PYTEST_REPLAY_OPTIONS=--replay-record-dir=${{ github.workspace }}\\pytest-replay --replay-base-name=Win-%CONDA_VERSION%-Py%PYTHON_VERSION%"
-          echo "##vso[task.setvariable variable=PYTEST_REPLAY_OPTIONS]%PYTEST_REPLAY_OPTIONS%"
-        shell: cmd
-
-      - name: Run Serial Tests
-        if: matrix.test-type == 'serial'
-        run: |
-          echo on
-          set PATH
-          call %UserProfile%\Miniconda3\condabin\activate.bat base||exit 1
-          set PATH
-          call conda install -y conda-verify||exit 1
-          set PERL=
-          set LUA=
-          set R=
-          pytest --color=yes -vv -n 0 --basetemp %UserProfile%\cbtmp_serial --cov conda_build --cov-report xml -m "serial" ${{ env.pytest-replay }}
-        shell: cmd
-
-      - name: Run Parallel Tests
-        if: matrix.test-type == 'parallel'
-        run: |
-          echo on
-          set PATH
-          call %UserProfile%\Miniconda3\condabin\activate.bat base||exit 1
-          set PATH
-          :: call conda remove -y conda-verify||exit 1
-          set PERL=
-          set LUA=
-          set R=
-          pytest --color=yes -vv -n auto --basetemp %UserProfile%\cbtmp --cov conda_build --cov-append --cov-report xml -m "not serial" ${{ env.pytest-replay }}
-        shell: cmd
+          allure_results: allure-results
+          gh_pages: gh-pages
+          allure_report: allure-report
+          allure_history: allure-history
+          keep_reports: 20
+      
+      - name: Deploy report to Github Pages
+        if: always()
+        uses: peaceiris/actions-gh-pages@v2
         env:
-          VS90COMNTOOLS: C:\Program Files (x86)\Common Files\Microsoft\Visual C++ for Python\9.0\VC\bin
-          LIB:
-
-      - name: Upload Pytest Replay
-        uses: actions/upload-artifact@v2
-        with:
-          path: ${{ github.workspace }}/pytest-replay
-          name: Win-${{ env.CONDA_VERSION }}-Python${{ matrix.python-version }}-${{ env. parameters.serial_or_parallel  }}
+          PERSONAL_TOKEN: ${{secrets.ACCESS_TOKEN}}
+          PUBLISH_BRANCH: gh-pages
+          PUBLISH_DIR: allure-history
+      
+      - name: Post the link to the report
         if: always()
+        uses: Sibz/github-status-action@v1
+        with: 
+            authToken: ${{secrets.GITHUB_TOKEN}}
+            context: 'Test report'
+            state: 'success'
+            sha: ${{ github.event.pull_request.head.sha }}
+            target_url: simple-elf.github.io/github-allure-history/${{ github.run_number }}
```

Comment:
> This seems to remove most of the CI setup, is this intentional?

This is only testing one piece so that I could make sure the allure report is working and showing the html generated report.. I have the rest to put back.. I could potentially only test these changes on Github itself using its CI builder hence the multiple runs..

Reply:
> What's that site doing? https://github.com/simple-elf seems like a random person?

That is trying to deploy the generated allure report to the gh page..  I will have to use a different method.. I able to locally generate the allure reports.. hence the trails..